README.md

```diff
@@ -135,7 +135,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [BI4ALL](https://www.bi4all.pt/en/) [:rocket:](https://www.linkedin.com/company/bi4all/jobs/) | Consulting services with several analytics solutions. | `Lisboa` `Porto` |
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa` <br> `Porto` `Tomar` <br> `Vila Real` `Viseu` |
 | [Eyenov](https://eyenov.com) | Computer Vision. Machine Learning. Mobile, desktop and embedded. | `Lisboa` `Remote` |
-| [DareData Engineering](https://daredata.engineering/)| Solving Data Science and Data Engineering problems for startups and enterprises. |`Remote`|
+| [DareData Engineering](https://www.daredata.engineering/join-the-team)| Data Science, Data Engineering and MLOps Consulting Boutique. | `Remote` |
```

Comment:
Sorry, I think I did not express myself correctly.
The company website was OK, I was suggesting adding that link to the carrers page (if you believe it is relevant).

Something like:

```suggestion
| [DareData Engineering](https://daredata.engineering/) [:rocket:](https://www.daredata.engineering/join-the-team) | Data Science, Data Engineering and MLOps Consulting Boutique. | `Remote` |
```

Reply:
Got it :)